src/missing-types.d.ts

```diff
@@ -327,9 +330,13 @@ declare module '@higlass/utils' {
     export function showMousePosition<T>(context: T, is2d?: boolean, isGlobal?: boolean): () => void;
     export function absToChr(
         absPosition: number,
-        chrInfo: ChromInfo<string>
+        chrInfo: Pick<ChromInfo, 'cumPositions' | 'chromLengths'>
     ): [chr: string, chrPositon: number, offset: number, insertPoint: number];
-    export function chrToAbs<Name>(chrom: Name, chromPos: number, chromInfo: ChromInfo<Name>): number;
+    export function chrToAbs<Name = string>(
+        chrom: Name,
+        chromPos: number,
+        chromInfo: Pick<ChromInfo<Name>, 'chrPositions'>
+    ): number;
```

Comment:
Unless these function implementations change, the use of `Pick` narrows the type of `ChromInfo` to just the essential fields so they are more reusable.

Reply:
I agree. Added to a previous PR, but we should use this in other Gosling.js codes (e.g., `assembly.ts`).